static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -206,11 +206,18 @@ const canShow = (): Promise<boolean> => {
             getVisitCount() >= params.minArticlesBeforeShowingBanner;
         const userAlreadyGivesUsMoney = userIsSupporter();
         const bannerIsBlockedForEditorialReasons = pageShouldHideReaderRevenue();
+        const showBecauseUserAlreadyGivesUsMoney =
```

Comment:
This is how the epic does it these days. Personally I find it clearer:

```javascript
    const isCompatibleUser = test.onlyShowToExistingSupporters
        ? userIsSupporter()
        : !userIsSupporter();
```

Reply:
👍 good shout, I've reused the function now

Reply:
(Note: what would you say to `onlyShowToExistingSupporters === userIsSupporter();`? Terser but arguably less clear, interested in your opinion)